js/index.js

```diff
@@ -1 +1,86 @@
 // Your code goes here
+const myInput1 = 
+document.querySelector('p');
+myInput1.addEventListener('click', (event) => {
+  myInput1.style.fontFamily = "Snell Roundhand, cursive";
+});
+
+const myInput2 = 
+document.querySelector('h1.logo-heading');
+myInput2.addEventListener('dblclick', (event) => {
+  myInput2.style.backgroundColor = "purple";
+});
+
+const myInput3 = 
+document.querySelector('h2');
+window.addEventListener('resize', (event) => {
+  myInput3.style.fontSize = "12px";
+  console.log(myInput3);
+});
+
+
+const myInput4 =
+document.querySelector("img");
+window.addEventListener('scroll', (event) => {
+  myInput4.style.border = "20px dotted pink";
+  console.log(myInput4);
+});
+
+const myInput5 =
+document.querySelector(".text-content");
+console.log(myInput5);
+window.addEventListener('mousemove', (event) => {
```

Comment:
if you do the eventListener on the window, then the mouse has to go all the off the window for the color change to take effect.  if you do it on the element itself, then it changes color when you move it off the element.  just FYI if you wanted it on the element instead.